.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: ruby
-rvm:
-  - 2.1
-  - 2.2
 before_install:
   - gem update bundler
-gemfile:
-  - gemfiles/fluentd_0_10.gemfile
-  - gemfiles/fluentd_0_12.gemfile
-  - gemfiles/fluentd_0_14.gemfile
+
+matrix:
+  include:
+    - rvm: 2.1
+      gemfile: gemfiles/fluentd_0_10.gemfile
```

Comment:
Fluentd v0.14 API does not work with Fluentd v0.10.
So we can drop old Fluentd version support.